content/pages/education/complaint-tool/form.md

```diff
@@ -10,7 +10,7 @@ description: Use your VA education benefits to pay for college or training progr
     <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/education/">Education</a></li>
-      <li><a aria-current="page" href="/education/complaint-tool/">Opt Out of Information Sharing</a></li>
+      <li><a aria-current="page" href="/education/complaint-tool/">GI Bill® School Complaint Tool</a></li>
```

Comment:
I'm always a little wary of using characters outside of the ASCII set directly in HTML. But if the curly single apostrophe works, this ® symbol should work as well. Is this a normal practice.

Otherwise we could use `&#174` instead.